src/styles/animation.scss

```diff
@@ -0,0 +1,10 @@
+@import './index.scss';
+
+.calendar__day {
+  transition: transform $transition-duration, background-color $transition-duration;
```

Comment:
Ensure that the variable `$transition-duration` is defined in the `index.scss` file or another imported SCSS file. Otherwise, this will cause an error during compilation.